wikipron/extract/core.py

```diff
@@ -11,6 +11,14 @@
     from wikipron.typing import Iterator, Pron
 
 
+def _skip_pron(pron: str) -> bool:
+    if "-" in pron:
+        return False
+    if " " in pron:
+        return False
+    return True
```

Comment:
Are the boolean values are flipped? For instance, if there is a dash, we want to return `True` (not `False`). The tests also seem to be backwards.